src/main/kotlin/no/nav/syfo/aktivitetskrav/database/AktivitetskravRepository.kt

```diff
@@ -39,26 +39,47 @@ class AktivitetskravRepository(private val database: DatabaseInterface) {
 
     fun createAktivitetskrav(
         aktivitetskrav: Aktivitetskrav,
-        previousAktivitetskravUuid: UUID?,
+        previousAktivitetskravUuid: UUID? = null,
+        referanseTilfelleBitUUID: UUID? = null,
+        connection: Connection? = null,
```

Comment:
Usikker på om dette er beste/peneste måten å gjøre det på, men siden kafka-consumeren som oppretter aktivitetskrav må bruke samme database-connection for alle records så sendes `connection` inn her. Hvis `connection` er null så åpnes en ny som også commiter.

Reply:
Kunne kafka-consumeren brukt en egen funksjon i `AktivitetskravService` som behandler en liste med `Aktivitetskrav`? Mått i så fall hatt en tilsvarende funksjon i `AktivitetskravRepository` Da hadde vi sluppet å sendt `Connection` på tvers av alle lagene og oppnådd _seperation of concerns_ i større grad? 😊

Reply:
Det som allerede er i denne PR'en er jo uansett bedre enn det det var, så vi kan jo eventuelt se på forbedringer som dette👆 senere 😊

Reply:
Ja, lurte på å gjøre noe sånt, men det gjøres og flere ting i samme connection i kafka-consumeren (henter eksisterende aktivitetskrav, evt oppdaterer stoppunkt), så ikke så lett å få til uten å legge mer logikk i repository. Så ja, enig i at vi får tenke litt på forbedringer i videre PRer.